examples/point_cloud/BuildReadMe.md

```diff
@@ -0,0 +1,29 @@
+
+```
+# Build PointCloud
+## Windows
+Build using the following commands
+```sh
+mkdir <ROOTDIR>/build
+cd <ROOTDIR>/build
+```
+Note:The location of this README is referred to as ROOTDIR  
```

Comment:
The build folder should not be next to this example. The idea of for the entire project is that whoever builds it should create a single build folder where everything gets build (sdk, examples, other tools or apps).
This way you won't need to add a new copy of dependencies (as you did for doc_opt).